admin/css/archive-picker.css

```diff
@@ -341,6 +341,7 @@
 .media-modal-content .picker-sidebar-header .close-sidebar {
   right: 35px;
   background: url("../../images/arrow-right.svg");
+  background-repeat: no-repeat;
```

Comment:
I'm aware that I'm nitpicking a bit, sorry about that 😅 but if you inspect in the editor you can see that the `background-repeat` property is actually already set somewhere else in the file, it's just overridden by `background`. So what I would prefer as a fix would be using the `background-image` and `background-color` properties, see below:

- Current:
<img width="428" alt="Screenshot 2024-05-23 at 10 37 41" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/9e0d7c23-499f-475f-9908-386dd8cf318c">

- Suggested:
<img width="427" alt="Screenshot 2024-05-23 at 10 38 33" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/90d9bf23-c606-4eed-a991-b8d8eadd44f2">

Let me know what you think 🙂